src/compile/selection/interval.ts

```diff
@@ -118,14 +118,22 @@ const interval:SelectionCompiler = {
       name: undefined,
       type: 'rect',
       encode: {
-        enter: {fill: {value: '#eee'}},
+        enter: {
```

Comment:
For the implementation, after we merge #2372, I think we can make these brush have some role names and add role configs to the `DEFAULT_CONFIG` in `config.ts` for these roles  and it should work.